### ISSUE TITLE
Generate pip package

### ISSUE THREAD

**Issue by kkazmierczyk** [IBM/javacore-analyser#12]
I would like simplify the usage of the tool by end user to avoid downloading the zip package.

The solution for this is to generate pip package to run it in the following way:
pip install javacore-analyser
`python -m javacore-analyser --input-data=/tmp/javacores --output-report=/tmp/javacore-analyser-report`

The alternative solution is to generate executable for each version which simplifies the usage of the tool (and end user does not require python). More information here: https://pyinstaller.org/en/stable/

**Comment by kkazmierczyk** [comment]
I started working on this according to https://packaging.python.org/en/latest/tutorials/packaging-projects/ and it requires rebuilding folder structure.
Once I have done it, the tests (which run from a different directory) fail with missing packages. I found, that this requires changing the declarations of imports:
https://stackoverflow.com/questions/4142151/how-to-import-the-class-within-the-same-directory-or-sub-directory

@PiotrAniola82 can we talk about that? I am not sure if I didn't miss anything.
